src/design-system/theme/useTheme.ts

```diff
@@ -0,0 +1,9 @@
+import { useContext } from "react";
+import { ThemeContext } from "styled-components";
+
+import { LightTheme } from "./lightTheme";
```

Comment:
Thinking out loud, this would be a good use case for having a default export in a `theme.ts` file.
We could check in it a Config and return either the light theme as a default or the theme set in the config.
Having a default export would allow us to not name it since we wouldn't know which one is selected

Reply:
I see the idea, but I think we should talk about it to be sure I get the full picture.

Reply:
Made the task